### ISSUE TITLE
In timeline, filter out dates we don't use somehow

### ISSUE THREAD

**Issue by evoskamp** [ggascoigne/amber#167]
It'd be nice to have a list of dates from config settings, but not display ones that have a magical value (1/1/1970, null) or are before a certain date or such.

**Comment by evoskamp** [comment]
Mmm, it seems the Welcome.tsx view should use a component to display with class deadline or deadlineExpired

        <li>
          <span className={classes.deadline}>
            Registration Open: <ConfigDate name='registrationOpen' format={MDY} />
          </span>
        </li>

**Comment by ggascoigne** [comment]
yeah, I thought about that, but we regularly left the date visible without emphasizing that it had expires for NW.  I was intentional, but perhaps it's not really such a good pattern.  It would be pretty simple to change.

**Comment by evoskamp** [comment]
I'm noodling with something, basically added a tab to settings with prefix config.timelineDates. Label, type date. Pull it into configuration so I can grab it wherever I need. That bit I have working and flipping it to an aray, sort, filter, I already have showing up fine.

There are already two styles (deadline, deadlineExpired), so I figure on writing a component that takes a config date to show/not show the card with the "if you access this after X date, contact ContactEmail" and can contextually show that, and then take all the dates in timelineDates, optionally check at a equal to or before date to filter out (e.g. 1/1/1970), though you could just delete them in config, and optionally the two classes to use for past and current/future dates.

Then whack all of the bottom half of Welcome for that component.

**Comment by evoskamp** [comment]
Like we dropped a few dates last year, dropped another date this year. And it makes it easy to create a role for that tab, rather than the explicit ones in Config.

Oh, and I use the name of the code entry as the label, as the database doesn't care about spaces, and other formatting for the code column.
